{{cookiecutter.project_slug}}/src/{{cookiecutter.package_name}}/oas3/{{cookiecutter.openapi_specs_version}}/components/schemas/fake.yaml

```diff
@@ -0,0 +1,29 @@
+FakeEnveloped:
```

Comment:
should be called EchoEnveloped and Echo no?

Reply:
I was thinking we could use the same api model for as a simple container for other experiments ...